### PR TITLE
Better reroll support and more rolling functions

### LINKED ISSUE #3: Reroll specific number of times or give it a list of values

### PR DESCRIPTION
**Pull request by Grissess**
- Closes #3

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -153,7 +154,7 @@ bot.on('message', msg => {
 			if (ret) {
 				msg.channel.send('Quote added!')
 			} else {
-				msg.channel.send('Error: incorrect format!\nFormat: `!quoteadd "<quote text>" -<name>`')
+        msg.channel.send('Error: incorrect format!\nFormat: `!quoteadd "<quote text>" -<name>`')
```
Comment (Grissess) [inline index.js:157]: The web UI definitely got me here.

After merging, you should definitely lint/format this. This part should be an easy catch.

Comment (reecer62) [reply index.js:157]: I'll lint this so hard

**Review by reecer62 — CHANGES_REQUESTED** [review]
Remove dead code

**Comment by Grissess** [comment]
Sorry, your request for changes must have gotten eaten--I only noticed it today :P

**Review by reecer62 — APPROVED** [review]

**Pull request merged**